packages/synapse-react-client/src/components/widgets/query-filter/CombinedRangeFacetFilter.tsx

```diff
@@ -45,24 +48,50 @@ export const CombinedRangeFacetFilter: React.FunctionComponent<
   onChange,
   collapsed = false,
 }: CombinedRangeFacetFilterProps) => {
+  const { getCurrentQueryRequest } = useQueryContext()
+
   const [isCollapsed, setIsCollapsed] = useState<boolean>(collapsed)
   const {
-    columnMin: col1Min,
+    columnName: col1Name,
```

Comment:
Do not use the column min from the current facet result, instead run a query asking for the facet stats without any selected facets.  Look for the facet associated to this facet column name